### PR TITLE
Fixed paranthesis spacing

### PR DESCRIPTION
**Pull request by ElifUskuplu**

### CODE REVIEW

File: src/set-theory/baire-space.lagda.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ baire-space = ℕ → ℕ
 is-set-baire-space : is-set baire-space
 is-set-baire-space f g =
   is-prop-all-elements-equal
-    λ p q →
+    ( λ p q →
       inv (isretr-eq-htpy p) ∙
```
Comment (EgbertRijke) [inline src/set-theory/baire-space.lagda.md:55]: Here we can also add parentheses around `inv (isretr-eq-htpy p)`. When parentheses are added, both arguments of path concatenation are treated equally, which makes the code more readable.

**Comment by EgbertRijke** [comment]
The code

```agda
is-set-baire-space : is-set baire-space
is-set-baire-space f g =
  is-prop-all-elements-equal
    ( λ p q →
      inv (isretr-eq-htpy p) ∙
      ( ap
         eq-htpy
           ( eq-htpy (λ n →
              eq-is-prop'
                ( is-set-ℕ (f n) (g n))
                ( htpy-eq p n)
                ( htpy-eq q n))) ∙
      isretr-eq-htpy q))
```

could be formatted as follows:

```
is-set-baire-space : is-set baire-space
is-set-baire-space f g =
  is-prop-all-elements-equal
    ( λ p q →
      ( inv (isretr-eq-htpy p)) ∙
      ( ( ap
          ( eq-htpy)
          ( eq-htpy
            ( λ n →
              eq-is-prop'
                ( is-set-ℕ (f n) (g n))
                ( htpy-eq p n)
                ( htpy-eq q n)))) ∙
        ( isretr-eq-htpy q)))
```

Note that there are a few differences: I put parentheses around the `ap` expression. This is a long argument of a path concatenation. I also put parentheses around `isretr-eq-htpy q`. This is the second argument of that same path concatenation. Since this is the second argument of the same operation, I put it at the same indentation level of the partentheses right before `ap` to show that these two argument have the same depth in the syntax tree of this definition. 

I also put `λ n →` on a new line. It was put on a new line because the lambda expression in its entirety is the argument of `eq-htpy`, which didn't fit on a line and therefore had to start on a new line.

Would you be able to format the definition below using the same conventions?

**Comment by ElifUskuplu** [comment]
> The code
> 
> ```agda
> is-set-baire-space : is-set baire-space
> is-set-baire-space f g =
>   is-prop-all-elements-equal
>     ( λ p q →
>       inv (isretr-eq-htpy p) ∙
>       ( ap
>          eq-htpy
>            ( eq-htpy (λ n →
>               eq-is-prop'
>                 ( is-set-ℕ (f n) (g n))
>                 ( htpy-eq p n)
>                 ( htpy-eq q n))) ∙
>       isretr-eq-htpy q))
> ```
> 
> could be formatted as follows:
> 
> ```
> is-set-baire-space : is-set baire-space
> is-set-baire-space f g =
>   is-prop-all-elements-equal
>     ( λ p q →
>       ( inv (isretr-eq-htpy p)) ∙
>       ( ( ap
>           ( eq-htpy)
>           ( eq-htpy
>             ( λ n →
>               eq-is-prop'
>                 ( is-set-ℕ (f n) (g n))
>                 ( htpy-eq p n)
>                 ( htpy-eq q n)))) ∙
>         ( isretr-eq-htpy q)))
> ```
> 
> Note that there are a few differences: I put parentheses around the `ap` expression. This is a long argument of a path concatenation. I also put parentheses around `isretr-eq-htpy q`. This is the second argument of that same path concatenation. Since this is the second argument of the same operation, I put it at the same indentation level of the partentheses right before `ap` to show that these two argument have the same depth in the syntax tree of this definition.
> 
> I also put `λ n →` on a new line. It was put on a new line because the lambda expression in its entirety is the argument of `eq-htpy`, which didn't fit on a line and therefore had to start on a new line.
> 
> Would you be able to format the definition below using the same conventions?

I've done it. Also, I'm checking countable-sets.lagda.md now.

**Comment by EgbertRijke** [comment]
Well done! I'll merge it when the checks are ready. Thanks so much Elif!

**Comment by EgbertRijke** [comment]
Ah, I'll wait for your changes in `countable-sets` :)

**Comment by EgbertRijke** [comment]
Very cool! That will be a great addition to the library!

**Pull request merged**